setup.cfg

```diff
@@ -70,6 +70,7 @@ openstack.metric.v1 =
     metric_measures_batch-metrics = gnocchiclient.v1.metric_cli:CliMetricsMeasuresBatch
     metric_measures_batch-resources-metrics = gnocchiclient.v1.metric_cli:CliResourcesMetricsMeasuresBatch
     metric_measures aggregation = gnocchiclient.v1.metric_cli:CliMeasuresAggregation
+    metric_aggregates_fetch = gnocchiclient.v1.metric_cli:CliAggregatesFetch
```

Comment:
metric_aggregates_fetch does seem slightly verbose now that i see it.

i think metric transform|translate|mutate are better. fetch seems weird to me now tbh.